uptycs/data_source_audit_configuration.go

```diff
@@ -0,0 +1,111 @@
+package uptycs
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/provider"
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/uptycslabs/uptycs-client-go/uptycs"
+)
+
+type dataSourceAuditConfigurationType struct {
+	p Provider
+}
+
+func (r dataSourceAuditConfigurationType) GetSchema(_ context.Context) (tfsdk.Schema, diag.Diagnostics) {
+	return tfsdk.Schema{
```

Comment:
Not a blocker, just a note, I think it would serve us well to add more validation checks on each of these attributes. This is definitely a nice to have future state

Reply:
The problem is all the validations are totally unknown and undocumented. I've added the ones i'm aware of in models.go but theres complexities in some like "if x then y's value has to be z" type stuff that are completely unknown to us as external contribs